src/ui/InputText/component.tsx

```diff
@@ -0,0 +1,7 @@
+function InputText() {
+  return (
+    <input type="text" placeholder="Ingresar tarea..."></input>
```

Comment:
the `input` element is a self-closing tag, please fix that

```diff
-<input type="text" placeholder="Ingresar tarea..."></input>
+<input type="text" placeholder="Ingresar tarea..." />
```

also, we should track the value of this element from a parent component, let's change the component to be more useful

```tsx
type InputTextProps = {
  onChange: (text: string) => void;
}

function InputText({ onChange }: InputTextProps) {
  return (
    <input
      type="text"
      placeholder="Ingresar tarea..."
      onChange={(event) => onChange(event.target.value)}
    />
```

let's see how that goes